Source/WebCore/rendering/ImageQualityController.cpp

```diff
@@ -152,9 +156,7 @@ InterpolationQuality ImageQualityController::chooseInterpolationQuality(Graphics
             return InterpolationQuality::Default;
     }
 
-    const AffineTransform& currentTransform = context.getCTM();
-    bool contextIsScaled = !currentTransform.isIdentityOrTranslationOrFlipped();
-    if (!contextIsScaled && size == imageSize) {
+    if (!contextIsScaled && scaledLayoutSize == scaledImageSize) {
```

Comment:
I think this can still compare the sizes before scaling. But why do we have to check whether the destinationSize is the same the image or not? The comment below say "There is no scale in effect..". I think we can say:
```
if (scaleFactor == { 1, 1 })
``` 
Or
```
if (image.size() == scaledImageSize)
```

Reply:
Done